po/en.po

```diff
@@ -101,32 +101,48 @@ msgstr "Directly Output Uppercase Letters"
 msgid "put_lowercase_to_buffer"
 msgstr "Put Lowercase Letters to Composing Buffer"
 
-#: src/McBopomofo.h:87
+#: src/McBopomofo.h:82
+msgid "disabled"
+msgstr "Disabled"
+
+#: src/McBopomofo.h:83
+msgid "input_bpmf_reading"
+msgstr "Input Bofomofo Readings"
```

Comment:
s/Input/Output/g? Same for the enum names and the zh translation…

Reply:
Fixed.